lichess-bot.py

```diff
@@ -222,6 +275,7 @@ def lichess_bot_main(li: lichess.Lichess,
                       "logging_level": logging_level}
 
     recent_bot_challenges: DefaultDict[str, List[Timer]] = defaultdict(list)
+    one_game = True
```

Comment:
I don't think this should be here.

Reply:
Good catch! I forgot to remove it after testing.